glossary.asciidoc

```diff
@@ -423,13 +418,13 @@ Satoshi Nakamoto::
     Satoshi Nakamoto is the name used by the person or group of people who designed Bitcoin and created its original reference implementation, Bitcoin Core. As a part of the implementation, they also devised the first blockchain database. In the process, they were the first to solve the double-spending problem for digital currency. Their real identity remains unknown.
 
 Script::
-    Bitcoin uses a scripting system for transactions. Forth-like, Script is simple, stack-based, and processed from left to right. It is purposefully not Turing-complete, with no loops.
+    Bitcoin uses a scripting system for transactions called Script. Forth-like, it is simple, stack-based, and processed from left to right. It is purposefully not Turing-complete, with no loops.
 
 ScriptPubKey (aka pubkey script)::
     ScriptPubKey or pubkey script, is a script included in outputs which sets the conditions that must be fulfilled for those satoshis to be spent. Data for fulfilling the conditions can be provided in a signature script.
 
 ScriptSig (aka signature script)::
-    ScriptSig or signature script, is the data generated by a spender which is almost always used as variables to satisfy a pubkey script.
+    ScriptSig or signature script is the data generated by a spender, which are almost always used as variables to satisfy a pubkey script.
```

Comment:
I think the subject of the sentence is a bit confusing. Shouldn't this all be singular? Like: "ScriptSig or signature script is the data generated by a spender, which is almost always used as a condition to satisfy a pubkey script."

Reply:
It is confusing.  What is used as the variables, the data (plural) or the ScriptSig (singular)?

My preference would be:
"ScriptSig or signature script is the data generated by a spender and it is almost always used as a condition to satisfy a pubkey script."

This keeps the focus on the ScriptSig and not on the data.

Reply:
agree with the suggestion by @Zendo102